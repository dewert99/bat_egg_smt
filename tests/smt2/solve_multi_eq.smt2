(declare-sort T)
(declare-const a T)
(declare-const b T)
(declare-const c T)
(declare-const d T)
(assert (= a b c d))
(check-sat)
(get-model)
(assert (not (= b d c)))
(check-sat)