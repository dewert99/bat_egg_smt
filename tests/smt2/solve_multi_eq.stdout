sat
(
 (define-fun a () T @T_3)
 (define-fun b () T @T_3)
 (define-fun c () T @T_3)
 (define-fun d () T @T_3)
)
unsat
